files/build_templates/sonic_debian_extension.j2

```diff
@@ -37,7 +37,6 @@ trap_push clean_sys
 sudo LANG=C chroot $FILESYSTEM_ROOT mount sysfs /sys -t sysfs
 
 sudo chroot $FILESYSTEM_ROOT service docker start
-sudo chroot $FILESYSTEM_ROOT docker version
```

Comment:
Docker version command gets information from docker engine via socket. Service runs docker engine but it doesn't wait until docker engine will be fully initialized. In some cases docker version command was run before docker was fully initialized. Actually we don't need in a build time information that docker version command returns.

I've run build of sonic-mellanox.bin target for 50 times with this patch and issue wasn't reproduced.